azurerm/internal/services/storage/storage_data_lake_gen2_filesystem_resource.go

```diff
@@ -73,13 +76,45 @@ func resourceStorageDataLakeGen2FileSystem() *schema.Resource {
 			},
 
 			"properties": MetaDataSchema(),
+
+			"ace": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Computed: true,
```

Comment:
we can remove this, since this shouldn't be configured unless a user opts to do so:

```suggestion
```